velox/type/tz/TimeZoneMap.cpp

```diff
@@ -39,6 +42,12 @@ inline std::chrono::minutes getTimeZoneOffset(int16_t tzID) {
   return std::chrono::minutes{(tzID <= 840) ? (tzID - 841) : (tzID - 840)};
 }
 
+const date::time_zone* locateZoneExt(std::string_view tz_name) {
```

Comment:
for consistency with other parts of the codebase, maybe call it `locateZoneImpl()`?